lib/tasks/boot.rake

```diff
@@ -18,12 +18,14 @@ namespace :boot do
   desc 'Tries to connect to Redis'
   task :redis do
     require Rails.root.join('app', 'lib', 'three_scale', 'redis_config')
-    require Rails.root.join('config', 'initializers', 'redis_hacks')
+    require Rails.root.join('app', 'lib', 'system', 'redis_pool')
```

Comment:
Also, while on this, we could fix RuboCop warnings :)
```
    require Rails.root.join('app/lib/three_scale/redis_config')
    require Rails.root.join('app/lib/system/redis_pool')
```

Reply:
Done